homeassistant/components/mikrotik/const.py

```diff
@@ -47,3 +51,114 @@
     "tx-rate",
     "uptime",
 ]
+
+MEGA = 1048576
+
+SENSOR_SYSINFO = "sysinfo"
+SENSOR_CPU = "cpu"
+SENSOR_MEMORY = "memory"
+SENSOR_DISK = "disk"
+SENSOR_DOWNLOAD_SPEED = "download_speed"
+SENSOR_UPLOAD_SPEED = "upload_speed"
+
+UNITS = ["bits-per-second", "byte", "memory", "space"]
+
+PARAM_SPEED = {"interface": MTK_DEFAULT_WAN, "duration": "1s"}
+ATTR_SYSINFO = [
+    "board-name",
+    "serial-number",
+    "version",
+    "factory-firmware",
+    "firmware-type",
+    "current-firmware",
+    "upgrade-frimware",
+    "routerboard",
+    "cpu",
+    "architecture-name",
```

Comment:
Static info is not allowed in state attributes except info critical to automations.

Please remove this sensor.

Reply:
Okay, removing the sysinfo sensor.